src/SemiclassicalOrthogonalPolynomials.jl

```diff
@@ -180,7 +180,10 @@ function semiclassical_jacobimatrix(Q::SemiclassicalJacobi, a, b, c)
         return Q.X
     elseif b == -one(eltype(Q.t))
         return semiclassical_jacobimatrix(Q.t, a, b, c)
-    end 
+    elseif Q.b == -one(eltype(Q.t))
+        newQ = SemiclassicalJacobi(Q.t, Q.a, one(Q.b), Q.c, Q)
```

Comment:
Why are we constructing with `one(Q.b)` instead of `zero(Q.b)`? Not saying it's wrong, just curious what the motivation is. What happens with this implementation if `b = 0`, does it hit some other special case?

Reply:
The case with `b = 1` is efficient (see Line 176). If we did `b = 0` it would have to be defined in terms of the `b = 1` case anyway since $P^{t, (a, -1, c)}$ is defined in terms of $P^{t, (a, 1, c)}$. I'll make a comment